src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/IncidentQuestion.kt

```diff
@@ -31,11 +32,12 @@ data class IncidentQuestion(
   @EmbeddedId
   val id: IncidentQuestionId,
 
-  @ManyToOne(fetch = FetchType.LAZY)
+  @ManyToOne
   @JoinColumn(name = "QUESTIONNAIRE_QUE_ID", updatable = false, nullable = false)
   val question: QuestionnaireQuestion,
 
   @OneToMany(mappedBy = "id.incidentQuestion", cascade = [CascadeType.ALL], orphanRemoval = true)
+  @Fetch(FetchMode.SUBSELECT)
```

Comment:
Adding the subselect (and other in this PR) reduced the number of selects (for one of the biggest queries) by over 150